web/src/routes/_rallyists/_withUserProvider/reward/submit.tsx

```
@@ -26,57 +36,120 @@ function Submit() {
 
-  const canSubmit = isEligible && !isPending && !isSuccess && !isError;
+  const canSubmit =
+    isStandardRewardObtainable && !isPending && !isSuccess && !isError;
 
   const handleSubmit = () => {
     mutate();
   };
 
-  const showUserInfos = user?.name || user?.email;
+  const standardRewards = [
+    <li>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)